reads2transcriptome/README.md

```diff
@@ -0,0 +1,3 @@
+# reads2transcriptome workflow test datasets
+
+This sub-directory contains test datasets for the https://github.com/Arcadia-Science/reads2transcriptome workflow.
```

Comment:
Can you add a sentence or two about where these files come from or why they're the ones that you're testing with?

Reply:
Done.